packages/vkui/src/components/UsersStack/UsersStack.tsx

```diff
@@ -18,11 +18,26 @@ const stylesDirection = {
   'column': styles['UsersStack--direction-column'],
 };
 
+interface Photo {
+  src: string;
+  renderWrapper?: (props: { children: React.ReactNode; key: string | number }) => React.ReactNode;
+}
```

Comment:
```suggestion
export type UsersStackRenderWrapperProps = {
  key: string | number;
  children: React.ReactElement;
  ['data-src']: string;
}

export type UsersStackPhotos = {
  src: string;
  renderWrapper?: (props: UsersStackRenderWrapperProps) => React.ReactNode;
}
```

1. вынес в пропсы функции в отдельный тип, т.к. нужно будет экспортировать для пользователя эти типы;
2. `children: React.ReactElement` – т.к. мы всегда элемент возвращаем
3.  `data-src: string` чтобы можно было в компоненте использовать если надо (не `src`, чтобы не путать пользователя, что ему надо самому `img` возвращать)
4. `interface` -> `type` скорее вкусовщина
5. нейминг можно будет поменять, просто добавил префикс, чтобы при экспорте было понятно про что это